nixos/doc/manual/installation/installing.xml

```diff
@@ -46,6 +46,12 @@
    to increase the font size.
   </para>
 
+  <para>
+    To install over a serial port connect with <literal>115200n8</literal>
+    (e.g. <command>picocom -b 115200 /dev/ttyUSB0</command>). When the
+    bootloader lists boot entries, select the serial console boot entry.
```

Comment:
I suppose this what we default to in `services.getty.serialSpeed`?

Reply:
Yes. https://github.com/NixOS/nixpkgs/issues/58198 has the list of different parts of boot that require setting a particular serial speed, including getty.